ros/src/computing/perception/localization/lib/autoware_lidar_localizer/include/autoware_lidar_localizer/util/convert_ros_msgs.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CONVERT_ROS_MSGS_H
+#define CONVERT_ROS_MSGS_H
+
+#include <tf/tf.h>
+
+#include <geometry_msgs/Pose.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/PoseWithCovarianceStamped.h>
+#include <geometry_msgs/TwistStamped.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <jsk_recognition_msgs/HistogramWithRange.h>
```

Comment:
Why is it necessary? I want to be independent of jsk msgs. 
It maybe cause trouble when upgrading

Reply:
@yukkysaito 
I made it to display the histogram of the nearest distance of map pointcloud and LiDAR pointcloud for visualization.
If you prefer not to use jsk msgs, I would like to remove this feature.

Reply:
removed